bridge_ui/config-overrides.js

```diff
@@ -0,0 +1,67 @@
+const { ProvidePlugin } = require("webpack");
+
+module.exports = function override(config, env) {
+  return {
+    ...config,
+    module: {
+      ...config.module,
+      rules: [
+        ...config.module.rules,
+        {
+          test: /\.js$/,
+          enforce: "pre",
+          use: ["source-map-loader"],
+        },
+        {
+          test: /\.wasm$/,
+          type: "webassembly/async",
+        },
+      ],
+    },
+    plugins: [
+      ...config.plugins,
+      new ProvidePlugin({
+        Buffer: ["buffer", "Buffer"],
+        process: "process/browser",
+      }),
+    ],
+    resolve: {
+      ...config.resolve,
+      fallback: {
```

Comment:
These are the polyfill packages that were used webpack 4: [Webpack 5 Node Polyfills Upgrade Cheatsheet
](https://gist.github.com/ef4/d2cf5672a93cf241fd47c020b9b3066a)